doc/openshift-logging-rhocp_4.17+.adoc

```diff
@@ -0,0 +1,108 @@
+# Application Logging on Red Hat OpenShift Container Platform (RHOCP) with Loki, Vector, and the RHOCP Cluster Observability Operator
+
+The following guide has been tested and is supported with Red Hat OpenShift Container Platform (RHOCP) version 4.17.
+
+Pod processes running in Kubernetes frequently produce logs. To effectively manage this log data and ensure no loss of log data occurs when a pod terminates, a log aggregation tool should be deployed on the Kubernetes cluster. Log aggregation tools help users persist, search, and visualize the log data that is gathered from the pods across the cluster. Log aggregation tools in the market today include:  EFK, LogDNA, Splunk, Datadog, IBM Operations Analytics, etc.  When considering log aggregation tools, enterprises make choices that are inclusive of their journey to cloud, both new cloud-native applications running in Kubernetes and their existing traditional IT choices.
+
+With the release of Logging 6.0 in RHOCP 4.17, the EFK stack is no longer supported. Logging 6.0 now manages log storage, collection, and visualization using Loki, Vector, and the Cluster Observability's new Logging UI Plugin. This guide describes the process of deploying the new stack using the Loki, RedHat OpenShift Logging, and Cluster Observability Operators. Use this stack to aggregate all container logs. After a successful installation, the Loki and log collector pods should reside inside the *openshift-logging* namespace of the cluster. Check out this link:++https://docs.openshift.com/container-platform/4.17/observability/logging/logging-6.0/log6x-upgrading-to-6.html[doc] for more information on the new changes introduced in Logging 6.0.
+
+## Prerequisites
+
+You must have existing object storage in order to configure LokiStack. Loki Operator supports AWS S3, Azure, GCS, Minio, OpenShift Data Foundation, and Swift as options for LokiStack object storage.
+
+Create a secret `logging-loki-s3` inside the `openshift-logging` namespace that contains the required fields needed for LokiStack to access your object storage. As an example, the command below creates a secret that allows access to an OpenShift Data Foundation managed S3 bucket running inside the cluster.
+
+[source,sh]
+----
+  oc create secret generic logging-loki-s3 \
+    --from-literal=access_key_id=key \
+    --from-literal=access_key_secret=secret \
+    --from-literal=bucketnames=bucket-name \
+    --from-literal=endpoint=endpoint \
+    --namespace openshift-logging
+----
+
+## Setup LokiStack and Cluster Log Forwarding 
+
+. Follow this link:++https://docs.openshift.com/container-platform/4.17/observability/logging/logging-6.0/log6x-about.html#quick-start[guide] to set up LokiStack and a ClusterLogForwarder in the `openshift-logging` namespace. When you create your LokiStack Custom Resource (CR) in step 2, replace the `storageClassName` definition with your desired storage class.
```

Comment:
rather than call out one thing (storageClassName), just note that you can customize the LokiStack as required - that could also be in the tips section